__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require('supertest');
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => db.end());
+/* beforeEach(() => seed(data)); */
+
+describe("GET /api/", () => {
+    describe("/topics", () => {
+        test("200: status code and contains the expected data type", async () => {
+            const { status, body } = await request(app).get("/api/topics");
+            const { topics } = body;
+            const expected = {
+                slug: expect.any(String),
+                description: expect.any(String)
+            };
+            expect(status).toBe(200);
+            topics.forEach((topics) => {
```

Comment:
don't forget to make sure that any arrays you're iterating over in a test aren't empty (ie. checking the length), otherwise an empty array will give you a false positive on this test